content/en/docs/instrumentation/python/benchmarks.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Benchmarks
+weight: 101
```

Comment:
```suggestion
weight: 190
```

Reply:
Should it be the same value as we have in js? https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/languages/js/benchmarks.md?plain=1#L3

Reply:
No, it should be 190 in the Python section (and probably in the JS section too, but that's another story). /cc @svrnm

Reply:
Fixed